packages/office-ui-fabric-react/src/components/Calendar/CalendarDay.tsx

```diff
@@ -391,16 +391,14 @@ export class CalendarDay extends React.Component<ICalendarDayProps, ICalendarDay
     }
   }
 
-  private _setDayRef(element: HTMLElement | null, day: IDayInfo, isNavigatedDate: boolean): void {
+  private _setDayCellRef(element: HTMLElement | null, day: IDayInfo, isNavigatedDate: boolean): void {
+    this.days[day.key] = element;
+
     if (isNavigatedDate) {
       this.navigatedDay = element;
     }
   }
 
-  private _setDayCellRef(element: HTMLElement | null, day: IDayInfo, isNavigatedDate: boolean): void {
-    this.days[day.key] = element;
-  }
-
```

Comment:
This is a little hard to read, but essentially the change is that the `<button>` used to have a separate ref for `this.navigatedDay`, used in the public `focus()` method. Now that focus goes to the `<td>` elements, I'm using `_setDayCellRef` to set both the `this.navigatedDay` ref and the existing `this.days` cell refs.